docs/excel/custom-functions-json.md

```diff
@@ -131,6 +131,7 @@ The `options` object enables you to customize some aspects of how and when Excel
 |  Property  |  Data type  |  Required  |  Description  |
 |:-----|:-----|:-----|:-----|
 |  `cancelable`  |  boolean  |  No<br/><br/>Default value is `false`.  |  If `true`, Excel calls the `onCanceled` handler whenever the user takes an action that has the effect of canceling the function; for example, manually triggering recalculation or editing a cell that is referenced by the function. If you use this option, Excel will call the JavaScript function with an additional `caller` parameter. (Do ***not*** register this parameter in the `parameters` property). In the body of the function, a handler must be assigned to the `caller.onCanceled` member. For more information, see [Canceling a function](custom-functions-web-reqs.md#canceling-a-function). |
+|  `requiresAddress`  | boolean | No <br/><br/>Default value is `false`. | <br /><br /> If `true`, the function will be passed the address of the cell that invoked your custom function. In this option, Excel will call the JavaScript function with the last parameter being an 'invocatationContext' paramter. In the body of the function, if `requiresAddress` is set, you can access `context.address` member. A function cannot be both streaming and require an address. For more information, see [Determine which cell invoked your custom function](https://docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-overview#determine-which-cell-invoked-your-custom-function). |
```

Comment:
@keyur32 can you change the text as follows if it is correct please: 

If true, your custom function can access the address of the cell that invoked your custom function. To get the address of the cell that invoked your custom function, use context.address in your custom function. For more information, see [Determine which cell invoked your custom function](https://docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-overview#determine-which-cell-invoked-your-custom-function). Custom functions cannot be set as both streaming and requiresAddress. When using this option, the 'invocatationContext' parameter must be the last parameter passed in options.

Reply:
```suggestion
|  `requiresAddress`  | boolean | No <br/><br/>Default value is `false`. | <br /><br /> If true, your custom function can access the address of the cell that invoked your custom function. To get the address of the cell that invoked your custom function, use context.address in your custom function. For more information, see [Determine which cell invoked your custom function](/office/dev/add-ins/excel/custom-functions-overview#determine-which-cell-invoked-your-custom-function). Custom functions cannot be set as both streaming and requiresAddress. When using this option, the 'invocationContext' parameter must be the last parameter passed in options. |
```